packages/location-field/src/styled.tsx

```diff
@@ -1,317 +0,0 @@
-import React from "react";
-import { useIntl } from "react-intl";
-import styled, { keyframes } from "styled-components";
-import { Spinner as FASpinner } from "@styled-icons/fa-solid/Spinner";
-
-export const HiddenContent = styled.span`
-  clip: rect(0, 0, 0, 0);
-  display: inline-block;
-  height: 0;
-  overflow: hidden;
-  width: 0;
-`;
-
-export const BaseButton = styled.button`
-  border: none;
-  background: none;
-`;
-
-export const Button = styled(BaseButton)`
-  color: #888;
-  margin: 0;
-  padding: 2px 5px;
-`;
-
-export const ClearBoth = styled.div`
-  clear: both;
-`;
-
-export const DropdownButton = styled(BaseButton)`
-  width: 30px;
-`;
-
-export const DropdownContainer = styled.span`
-  position: contents;
-  width: 100%;
-`;
-
-export const MenuItemList = styled.ul.attrs({
-  role: "listbox"
-})`
-  background-clip: padding-box;
-  background-color: #fff;
-  border-radius: 4px;
-  border: 1px solid rgba(0, 0, 0, 0.15);
-  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
-  float: left;
-  font-size: 14px;
-  left: 0;
-  list-style: none;
-  margin: 2px 0 0;
-  min-width: 160px;
-  padding: 5px 0;
-  position: absolute;
-  text-align: left;
-  top: 100%;
-  /* this is an annoyingly high number, but is needed to be on top of some otp-rr components */
-  z-index: 1000000;
-`;
-
-export const Dropdown = ({
```

Comment:
🚨🚨🚨I'm not sure we're able to do this. I'm pretty sure I recall trimet or someone else requiring these components to be exported without styles!

Reply:
Can they just remain on the current version? Breaking changes are not automatically upgraded.

Reply:
What happens if they want a new feature though

Reply:
Per @grant-humphries on the other discussion thread, they are not using this component, so we should be good. Commit https://github.com/opentripplanner/otp-ui/pull/533/commits/c1877f10ab31f23e188785590efa92beb659256c has the breaking change for the `Dropdown` removal.